README.md

```diff
@@ -44,38 +37,8 @@ command line like this:
 ./normalizer < sample.csv > output.csv
 ```
 
-Normalized, in this case, means:
+### The problem: [Display data from an API for human consumption](BROWSER_README.md)
 
-* The entire CSV is in the UTF-8 character set.
-* The `Timestamp` column should be formatted in RFC3339 format.
-* The `Timestamp` column should be assumed to be in US/Pacific time;
-  please convert it to US/Eastern.
-* All `ZIP` codes should be formatted as 5 digits. If there are less
-  than 5 digits, assume 0 as the prefix.
-* The `FullName` column should be converted to uppercase. There will be
-  non-English names.
-* The `Address` column should be passed through as is, except for
-  Unicode validation. Please note there are commas in the Address
-  field; your CSV parsing will need to take that into account. Commas
-  will only be present inside a quoted string.
-* The `FooDuration` and `BarDuration` columns are in HH:MM:SS.MS
-  format (where MS is milliseconds); please convert them to the
-  total number of seconds expressed in floating point format.
-  You should not round the result.
-* The `TotalDuration` column is filled with garbage data. For each
-  row, please replace the value of `TotalDuration` with the sum of
-  `FooDuration` and `BarDuration`.
-* The `Notes` column is free form text input by end-users; please do
-  not perform any transformations on this column. If there are invalid
-  UTF-8 characters, please replace them with the Unicode Replacement
-  Character.
+Please write a static webpage that loads data from `https://swapi.dev/api/planets/` and displays that data in an HTML `table`. Name the entrypoint for your application `index.html`.
```

Comment:
```suggestion
Please write a static webpage that loads data from a specified public API, and formats & displays that data in HTML. You may use libraries, frameworks and/or build tools as long as clear, simple instructions are provided for how to run them. Please ensure your solution can be easily viewed in the browser.
```

Reply:
my suggestion here is to make these descriptions a little more generalized, so that if we end up altering them in the problem READMEs we don't have to worry as much about keeping them consistent here. Also from the perspective of a candidate reading this for the first time, the specificity felt a little bit strange.